spec/core_functions/string/to_upper_case.hrx

```diff
@@ -0,0 +1,121 @@
+<===> empty/input.scss
+a {b: to-upper-case("")}
+
+<===> empty/output.css
+a {
+  b: "";
+}
+
+<===>
+================================================================================
+<===> alphabet/input.scss
+a {b: to-upper-case("abcdefghijklmnopqrstuvqxyz")}
+
+<===> alphabet/output.css
+a {
+  b: "ABCDEFGHIJKLMNOPQRSTUVQXYZ";
+}
+
+<===>
+================================================================================
+<===> number/input.scss
+a {b: to-upper-case("1234567890")}
+
+<===> number/output.css
+a {
+  b: "1234567890";
+}
+
+<===>
+================================================================================
+<===> non_ascii/input.scss
+// Only ASCII characters have their case changed.
+a {b: to-upper-case("äçðøþ")}
+
+<===> non_ascii/output.css
+@charset "UTF-8";
+a {
+  b: "äçðøþ";
+}
+
+<===>
+================================================================================
+<===> unquoted/input.scss
+a {b: to-lower-case(aBcDeF)}
```

Comment:
Swap this and the test below with the `to-upper-case` tests in `to_lower_case.hrx`